homeassistant/backup_restore.py

```diff
@@ -0,0 +1,95 @@
+"""Home Assistant module to handle restoring backups."""
+
+import json
+import logging
+import os
+import shutil
+import sys
+from tempfile import TemporaryDirectory
+from typing import TypedDict
+
+import securetar
+
+RESTORE_BACKUP_FILE = ".HA_RESTORE"
+KEEP_PATHS = ("backups", ".HA_RESTORE")
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class RestoreBackupFileContent(TypedDict):
```

Comment:
The idea is to expand this to also support passwords in a follow-up PR, so while this is not needed as a dict now, it makes future enhancement simpler.

Reply:
I'd make this a dataclass instead.